master/pkg/schemas/expconf/storage_config.go

```diff
@@ -122,6 +123,16 @@ func (s SharedFSConfigV0) PathInContainer() string {
 	return filepath.Join(DefaultSharedFSContainerPath, *s.RawStoragePath)
 }
 
+// PathInContainerOrHost returns the PathInContainer if it exists,
+// otherwise returns RawHostPath if available.
+func (s SharedFSConfigV0) PathInContainerOrHost() string {
+	path := s.PathInContainer()
+	if _, err := os.Stat(path); os.IsNotExist(err) && s.RawHostPath != nil {
+		return *s.RawHostPath
```

Comment:
Why do you need to use the Raw API here?  Why not just return s.HostPath() if PathInContainer() doesn't exist?

Reply:
My main concern is with causing a panic in the server. I switched it to use `s.HostPath()` but it would feel a lot safer if HostPath used a default instead of panic.

Reply:
actually, updated the code to return an error instead